src/app/app.module.ts

```diff
@@ -62,41 +63,45 @@ const appRoutes: Routes = [
     VoteToPercentPipe,
     TopBilledCastCardComponent
   ],
-	imports: [
-		BrowserModule,
-		HttpClientModule,
-		FormsModule,
-		AngularSvgIconModule.forRoot(),
-		RouterModule.forRoot(appRoutes),
-		BrowserAnimationsModule,
-		MatSlideToggleModule,
-		MatCardModule,
-		MatGridListModule,
-		// NgCircleProgressModule,
-		NgCircleProgressModule.forRoot({
-			// set defaults here
-			radius: 100,
-			outerStrokeWidth: 16,
-			innerStrokeWidth: 8,
-			outerStrokeColor: "#78C000",
-			innerStrokeColor: "#C7E596",
-			animationDuration: 300
-		}),
-		YouTubePlayerModule,
-		MatInputModule,
+  imports: [
+    BrowserModule,
+    HttpClientModule,
+    FormsModule,
+    AngularSvgIconModule.forRoot(),
+    RouterModule.forRoot(appRoutes),
+    BrowserAnimationsModule,
+    MatSlideToggleModule,
+    MatCardModule,
+    MatGridListModule,
+    // NgCircleProgressModule,
+    NgCircleProgressModule.forRoot({
+      // set defaults here
+      radius: 100,
+      outerStrokeWidth: 16,
+      innerStrokeWidth: 8,
+      outerStrokeColor: "#78C000",
+      innerStrokeColor: "#C7E596",
+      animationDuration: 300
+    }),
+    YouTubePlayerModule,
+    MatInputModule,
     ReactiveFormsModule,
     MaterialModule,
     MatNativeDateModule,
     FormsModule,
     ToastrModule.forRoot(),
-    MatButtonModule
-	],
+    MatButtonModule,
+    CustomerFeatureAModule
+  ],
   providers: [
     DataService, {
       provide: HTTP_INTERCEPTORS,
       useClass: ParamInterceptor,
       multi: true
     }],
+  exports: [
+    GenreComponent
```

Comment:
Try to get rid of components in App module. Ideally, app module should only import other modules

Reply:
ok, rid off export components in App module